ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -884,6 +908,23 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: habana-jupyter-datascience-ubi8-python-3.8-test-e2e
+      cli: latest
+      commands: |
+        oc new-project habana-jupyter-datascience-ubi8-python-3-8
+        IFS=':' read -r -a NOTEBOOK_IMAGE <<< "${JUPYTER_TRUSTYAI_IMAGE}"
```

Comment:
A small fix on the env name.
${JUPYTER_TRUSTYAI_IMAGE} -> ${HABANA_JUPYTER__DATA_SCIENCE_IMAGE}